### ISSUE TITLE
exp: temp dir runs ignore `.dvc/config.local`

### ISSUE THREAD

**Issue by dberenbaum** [iterative/dvc#7490]
Hello, when running experiments in parallel, I am finding that local cache settings are ignored.

I am using dvc 2.9.2.

when I am in my repo:
```
(venv) [me@computer folder]$ dvc config -l
remote.remote.url=//remote/url
core.remote=remote
remote.remote.url=//local/remote/url
cache.dir=/cache/dir
cache.type=hardlink,symlink
```

but when I cd to one of the temporary experiment directories:
```
(venv) [me@computer tmp0zpspb9w]$ dvc config -l
remote.almds.url=//remote/url
core.remote=remote
cache.dir=/cache/dir
```
I mentioned this on Discord and was told that this could be relevant to this issue.

_Originally posted by @gregstarr in https://github.com/iterative/dvc/issues/5800#issuecomment-1074602366_

**Comment by dberenbaum** [comment]
@pmrowla Any thoughts on how to handle this?

**Comment by pmrowla** [comment]
This is a known issue (duplicate of https://github.com/iterative/dvc/issues/7254)

Right now we generate our own `config.local` in the temp dir so that we can configure the cache directory to point to the correct location (and we don't propagate anything else from the parent repo's `config.local`)

**Comment by dberenbaum** [comment]
Thanks @pmrowla! Closing as a duplicate.
